birdhouse/pavics-compose.sh

```diff
@@ -85,30 +81,62 @@ for adir in $AUTODEPLOY_EXTRA_REPOS; do
 done
 export AUTODEPLOY_EXTRA_REPOS_AS_DOCKER_VOLUMES
 
-# we apply all the templates
-find $ALL_CONF_DIRS -name '*.template' |
-  while read FILE
-  do
-    DEST=${FILE%.template}
-    cat ${FILE} | envsubst "$VARS" | envsubst "$OPTIONAL_VARS" > ${DEST}
-  done
+BUILD_DIR="${BUILD_DIR:-"${COMPOSE_DIR}/build"}"
```

Comment:
Sorry for nitpicking but I'd rather prefer this default value appear in `birdhouse/default.env` rather than here so it is more easily discoverable by our users.

I know you already documented in `env.local.example` but the easier any configs, in general, can be found, the better.